infra-as-code/bicep/modules/privateDnsZones/README.md

```diff
@@ -0,0 +1,120 @@
+
+> For the examples below we assume you have downloaded or cloned the Git repo as-is and are in the root of the repository as your selected directory in your terminal of choice.
+
+### Azure CLI
+```bash
+# For Azure global regions
+# Set Platform connectivity subscription ID as the the current subscription 
+ConnectivitySubscriptionId="[your platform management subscription ID]"
+az account set --subscription $ConnectivitySubscriptionId
+
+az group create --location eastus \
+   --name Hub_PrivateDNS_POC
+
+az deployment group create \
+   --resource-group Hub_PrivateDNS_POC  \
+   --template-file infra-as-code/bicep/modules/privateDnsZones/privateDnsZones.bicep \
+   --parameters @infra-as-code/bicep/modules/privateDnsZones/privateDnsZones.parameters.example.json
+```
+OR
+```bash
+# For Azure China regions
+# Set Platform connectivity subscription ID as the the current subscription 
+ConnectivitySubscriptionId="[your platform management subscription ID]"
+az account set --subscription $ConnectivitySubscriptionId
+
+az group create --location chinaeast2 \
+   --name Hub_PrivateDNS_POC
+
+az deployment group create \
+   --resource-group Hub_PrivateDNS_POC  \
+   --template-file infra-as-code/bicep/modules/privateDnsZones/privateDnsZones.bicep \
+   --parameters @infra-as-code/bicep/modules/privateDnsZones/mc-privateDnsZones.parameters.example.json
+```
+
+### PowerShell
+
+```powershell
+# For Azure global regions
+# Set Platform connectivity subscription ID as the the current subscription 
+$ConnectivitySubscriptionId = "[your platform management subscription ID]"
+
+Select-AzSubscription -SubscriptionId $ConnectivitySubscriptionId
+
+New-AzResourceGroup -Name 'Hub_PrivateDNS_POC' `
+  -Location 'eastus'
+  
+New-AzResourceGroupDeployment `
+  -TemplateFile infra-as-code/bicep/modules/privateDnsZones/privateDnsZones.bicep `
+  -TemplateParameterFile infra-as-code/bicep/modules/privateDnsZones/privateDnsZones.parameters.example.json `
+  -ResourceGroupName 'Hub_PrivateDNS_POC'
+```
+OR
+```powershell
+# For Azure China regions
+# Set Platform connectivity subscription ID as the the current subscription 
+$ConnectivitySubscriptionId = "[your platform management subscription ID]"
+
+Select-AzSubscription -SubscriptionId $ConnectivitySubscriptionId
+
+New-AzResourceGroup -Name 'Hub_PrivateDNS_POC' `
+  -Location 'chinaeast2'
+  
+New-AzResourceGroupDeployment `
+  -TemplateFile infra-as-code/bicep/modules/privateDnsZones/privateDnsZones.bicep `
+  -TemplateParameterFile infra-as-code/bicep/modules/privateDnsZones/mc-privateDnsZones.parameters.example.json
+  -ResourceGroupName 'Hub_PrivateDNS_POC'
+```
+## Example Output in Azure global regions
+
+![Example Deployment Output](media/privateDnsZonesExampleDeploymentOutput.png "Example Deployment Output in Azure global regions")
+
+## Example Output in Azure China regions
+![Example Deployment Output](media/mc-privateDnsZonesExampleDeploymentOutput.png "Example Deployment Output in Azure China")
```

Comment:
Unfortunately, I don't have access to an Azure China region and can't show a sample deployment here.

Reply:
We can add the Azure China output example in a follow-on effort.

Reply:
done